packages/discovery-react-components/src/components/DocumentPreview/utils/box.ts

```diff
@@ -0,0 +1,30 @@
+import { CellPage } from '../types';
+import { ProcessedBbox } from '../../../utils/document/processDoc';
+
+/**
+ * Check whether two bbox intersect
+ * @param boxA first bbox
+ * @param boxB second bbox
+ * @returns bool
+ */
+function intersects(boxA: number[], boxB: number[]): boolean {
+  const [leftA, topA, rightA, bottomA, pageA] = boxA;
+  const [leftB, topB, rightB, bottomB, pageB] = boxB;
+  return !(leftB > rightA || rightB < leftA || topB > bottomA || bottomB < topA || pageA != pageB);
+}
```

Comment:
Add a test for this, please. Check our old code; we may have already written one.

Reply:
I thought I already had a test for this in `box.test.ts`